modules/test/playwright/tests/frontend-taglib-clay/main/select.spec.ts

```diff
@@ -0,0 +1,67 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2025 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import {expect, mergeTests} from '@playwright/test';
+
+import {claySamplePageTest} from './fixtures/claySamplePageTest';
+import {TabName} from './pages/ClaySamplePage';
+
+const test = mergeTests(claySamplePageTest);
+
+test.beforeEach(async ({claySamplePage}) => {
+	await claySamplePage.selectTab(TabName.FORM_ELEMENTS);
+});
+
+test('Asserts selectors are working correctly', async ({page}) => {
```

Comment:
We should rename this test to something like "Asserts Clay's select elements are working correctly".